site/documentation/content/concepts/device-identity.md

```diff
@@ -147,3 +147,34 @@ After having verified the client certificate using the trust anchor(s), the prot
 the client certificate's *subject DN* and invokes the Credentials API's
 [*get Credentials*]({{< relref "/api/credentials#get-credentials" >}}) operation in order to retrieve the
 [*x509-cert* type credentials]({{< relref "/api/credentials#x509-certificate" >}}) that are on record for the device.
+
+### JSON Web Token based Authentication
+
+The MQTT protocol adapter supports authentication of devices with a JSON Web Token (JWT) based mechanism. In this case,
+the protocol adapter verifies the JWT by using the public key / certificate that is on record for the device in the
+device registry. Further more does the protocol adapter check if the JWT is valid. To be valid the JWT has to provide
+the following information in its header and payload.
+
+The JWT header has to provide two fields that indicate the type of token (`typ`) and the signing algorithm (`alg`).
+Both fields are mandatory and the type has to be `JWT`. For the signing algorithm `RS256` and `ES256` are supported.
```

Comment:
FMPOV we do not need to restrict the signing algorithms to these two only. Instead, we should probably follow the guidelines in [RFC 8725](https://datatracker.ietf.org/doc/html/rfc8725#name-use-appropriate-algorithms) and also support stronger algorithms.

Reply:
Sounds reasonable. So I would propose we try to extend this to RS384, RS512, ES384 and ES512: https://www.rfc-editor.org/rfc/rfc7518#section-3

Reply:
Very good idea! I added the possibility to also use the stronger variants of the signing algorithms.